### PR TITLE
[6.15] negative test for webhook task

### PR DESCRIPTION
**Pull request by pondrejk**
### Problem Statement
The "Deliver webhook..." task for the triggered hook previously succeeded even If the webhook execution failed for some reason. This will change in 6.15

### Solution
I'm adding a negative test to cover this scenario. Note that the upstream PR isn't merged yet, so prt trigger would fail atm

### Related Issues
SAT-21112

### CODE REVIEW

File: tests/foreman/api/test_webhook.py [hunk 1]
```diff
@@ -170,3 +170,27 @@ def test_positive_event_triggered(self, module_org, target_sat):
             shell.send('foreman-tail')
             repo.sync()
             assert_event_triggered(shell, hook.event)
+
+    @pytest.mark.tier1
```
Comment (jameerpathan111) [inline tests/foreman/api/test_webhook.py:174]:
```suggestion
```
We don't need these markers anymore, right?

File: tests/foreman/api/test_webhook.py [hunk 2]
```diff
@@ -170,3 +170,27 @@ def test_positive_event_triggered(self, module_org, target_sat):
...
+    @pytest.mark.parametrize('setting_update', ['safemode_render=False'], indirect=True)
+    def test_negative_event_task_failed(self, module_org, target_sat, setting_update):
+        """Create a webhook with unreachable targete and assert the associated task
```
Comment (jameerpathan111) [inline tests/foreman/api/test_webhook.py:177]:
```suggestion
        """Create a webhook with unreachable target and assert the associated task
```

File: tests/foreman/api/test_webhook.py [hunk 3]
```diff
@@ -170,3 +170,27 @@ def test_positive_event_triggered(self, module_org, target_sat):
...
+        failed
+
+        :id: d4a49556-9413-46e8-bcb5-7afd0184bdb2
+
+        :expectedresults: Deliver webhook task fails
+
+        :CaseImportance: High
+        """
+        hook = entities.Webhooks(
```
Comment (jameerpathan111) [inline tests/foreman/api/test_webhook.py:186]:
```suggestion
        hook = target_sat.api.Webhooks(
```
Same suggestion for other occurrences of `entities`.

**Review by jameerpathan111 — CHANGES_REQUESTED** [review]

**Comment by pondrejk** [comment]
@jameerpathan111 changes committed, thanks!

**Review by Griffin-Sullivan — COMMENTED** [review]
Where are we asserting that the task failed? Also are you able to get local results with a packit installed stream Satellite?

**Comment by pondrejk** [comment]
```
pytest tests/foreman/api/test_webhook.py -k negative_event_task_failed
========================================== test session starts ===========================================
                                                      

tests/foreman/api/test_webhook.py .                                                                [100%]

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_webhook.py -k  test_negative_event_task_failed

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_webhook.py -k test_negative_event_task_failed

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_webhook.py -k test_negative_event_task_failed

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_webhook.py -k test_negative_event_task_failed

**Comment by pondrejk** [comment]
@Griffin-Sullivan @jameerpathan111 the thing landed in stream, prt now passes

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**